manifests/a/AnyTXT/AnyTXTSearcher/1.2.540/AnyTXT.AnyTXTSearcher.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: AnyTXT.AnyTXTSearcher
+PackageVersion: 1.2.540
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://anytxt.net/release/AnyTXT.Searcher.1.2.540.exe
+  InstallerSha256: 4758E770122B1160625323F58967125E2F7FE72887F6EC7982F3065690F04641
+  InstallModes:
+  - silent
+  InstallerSwitches:
+    Silent: /VERYSILENT

```

Comment:
```suggestion
```
winget-cli runs all inno installers with `/silent` by default.

If a prompt cannot be suppressed with `/silent`, you can use `/verysilent` or try using `/suppressmsgboxes`, but this isn't needed for all inno installers.

Reply:
Hi, thanks for the response.

While running the "normal" winget install command with this manifest `winget install --manifest <manifest to anytxt that I created>`, there's a GUI window indicating current installation progress, but dissappears automatically when it completes, I'm not sure if this is allowed by winget, that's why I added the `/verysilent` switch into the manifest.

![image](https://user-images.githubusercontent.com/24511967/147414668-e4d4ffd4-620f-48b1-847b-c8c891f93828.png)

Above image is a screenshot of the window, it says "extracting files: (file path)", albeit in Chinese. It dissapears without user intervention after the progress bar completes, and winget commandline says installation complete. I'm not entirely sure if this is the behaviour expected by WinGet.

Reply:
Progress bars are allowed to be in WinGet and it's also the default experience which allows the user to see what's happening and whether the install is frozen or not.

I just did `winget install --manifest "C:\Users\WinGet\Documents\GitHub\winget-pkgs\manifests\a\AnyTXT\AnyTXTSearcher\1.2.540" --silent` and I wasn't able to see any progress bars because the `--silent` switch means `/VERYSILENT`.

The only thing that isn't allowed to be in WinGet at the moment is user intervention prompts, such as driver prompts.

Reply:
Thank you very much. I'm now committing your suggested changes and resolving this conversation.